src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -48,8 +51,9 @@ class ProfilerController
      * @param string                       $toolbarPosition The toolbar position (top, bottom, normal, or null -- use the configuration)
      * @param ContentSecurityPolicyHandler $cspHandler      The Content-Security-Policy handler
      * @param string                       $baseDir         The project root directory
+     * @param KernelInterface              $kernel          The kernel
```

Comment:
Hmm. IMHO it could be the time to remove the whole docblock instead 😄 (it brings nothing, appart from the `$toolbarPosition` description, which could go to the property). But that's still for lower branches.